### ISSUE TITLE
Typo in number causes the whole grid to not render (numbro.js)

### ISSUE THREAD

**Issue by aarondfrancis** [handsontable/handsontable#7047]
### Description
If a user accidentally types e.g. `- 47` (note the space) then the whole grid ceases to work, because numbro.js throws an exception.

### Steps to reproduce
1. View the demo, where `- 47` is already set.
2. Alternatively, open the demo, comment out the bad data and type `- 47` for yourself.

### Demo
https://jsfiddle.net/tqgxerv5/

### Your environment
* Handsontable version: 8.0.0-beta1
* Browser Name and version: Chrome Latest
* Operating System: OSX

In the meantime, I've created my own numeric renderer as follows:

```js
import Handsontable from 'handsontable';

function render(instance, td, row, col, prop, value, cellProperties) {
    // Some values completely crash numbro.js and cause the whole sheet
    // to not render, so we wrap the numeric renderer in a try/catch.
    try {
        Handsontable.renderers.NumericRenderer.apply(this, [instance, td, row, col, prop, value, cellProperties]);
    } catch (e) {
        Handsontable.renderers.TextRenderer.apply(this, [instance, td, row, col, prop, value, cellProperties]);
    }
}

export default render;

```

**Comment by AMBudnik** [comment]
That's an interesting finding. Thank you for sharing @aarondfrancis 

Just to confirm - it works the same in `8.0.0-beta2-rev16` https://jsfiddle.net/oL1hv4ru/ and in the last non-beta release `7.4.2` https://jsfiddle.net/23fuv5ar/

**Comment by AMBudnik** [comment]
Hi @aarondfrancis 

We just released Handsontable v15.0.0, where the issue seems to be fixed. Among the list of changes, we also bumped Numbro to `2.5.0`. Reference https://github.com/handsontable/handsontable/pull/10929. 

Thank you again for your feedback.
